draft-ietf-jsonpath-base.md

```diff
@@ -527,7 +528,7 @@ Consider this example. With the argument `{"a":[{"b":0},{"b":1},{"c":2}]}`, the
 query `$.a[*].b` selects the following list of nodes: `0`, `1`
 (denoted here by their value).
 
-The query consists of `$` followed by three selectors: `.a`, `[*]`, and `.b`.
+The query consists of `$` followed by three appenders: `.a`, `[*]`, and `.b`.
 
 Firstly, `$` selects the root node which is the argument.
```

Comment:
Talking of `$` as if it was a selector doesn't seem consistent with "root identifier" language.